clients/privacy-center/__tests__/common/geolocation.test.ts

```diff
@@ -137,6 +172,14 @@ describe("getGeolocation", () => {
       expect(geolocation).toBeNull();
     });
 
+    it("ignores invalid three-character country codes in geolocation query param", async () => {
+      const req = createRequest({
+        url: "https://privacy.example.com/fides.js?geolocation=USA",
+      });
+      const geolocation = await lookupGeolocation(req);
+      expect(geolocation).toBeNull();
```

Comment:
Is there not a reason to test `USA-NY` also, since that was removed above? Or do you think that's a redundant test?

Reply:
Nope, more tests are totally reasonable! If a second engineer thinks a test helps I always prefer to add it - modifying now...